infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -64,13 +67,13 @@ var additionalSettings = {
   WEBSITES_ENABLE_APP_SERVICE_STORAGE: 'false'
 }
 
-var nodeEnv = environment == 'dev' ? {NODE_ENV: 'development'} : {}
+var nodeEnv = nodeDeveloperMode ? { NODE_ENV: 'development' } : {}
 
 var appSettings = union(settings, secureSettings, additionalSettings, additionalSecureSettings, nodeEnv)
 
-var functionAcbsName = 'tfs-${environment}-function-acbs'
-var privateEndpointName = 'tfs-${environment}-function-acbs'
-var applicationInsightsName = 'tfs-${environment}-function-acbs'
+var functionAcbsName = 'tfs-${environment}-${resourceNameFragment}'
+var privateEndpointName = 'tfs-${environment}-${resourceNameFragment}'
+var applicationInsightsName = 'tfs-${environment}-${resourceNameFragment}'
```

Comment:
I've kept these as three different variables. However, as they are all identical (for all use cases), it seems reasonable to make them just a single one. Happy to make the change if you wish!